ui/src/logs/utils/table.ts

```diff
@@ -157,3 +157,14 @@ export const applyChangesToTableData = (
     values: updatedValues,
   }
 }
+
+export const isEmptyInfiniteData = (data: {
+  forward: TableData
+  backward: TableData
+}) => {
+  return isEmptyTableData(data.forward) && isEmptyTableData(data.backward)
```

Comment:
a little confused by this naming, does this replace the `isTableEmpty()` from above?